skyportal/tests/frontend/test_followup_requests.py

```diff
@@ -79,12 +79,30 @@ def add_allocation(instrument_id, group_id, token):
     return data["data"]
 
 
-def add_followup_request_using_frontend_and_verify(
+def add_allocation_lt(instrument_id, group_id, token):
```

Comment:
Can you add a line here that checks if the LT queue listener is online (see `tests/api/followup_requests.py` for an example of how to do this for SEDM). If the LT queue listener is down, can you skip the LT tests here ?